.github/workflows/nv-a6000.yml

```diff
@@ -23,7 +23,7 @@ jobs:
   unit-tests:
     runs-on: [self-hosted, nvidia, a6000]
     container:
-      image: nvcr.io/nvidia/pytorch:23.03-py3
+      image: nvcr.io/nvidia/pytorch:23.09-py3
```

Comment:
Updating container to have pydantic 2+ support and updated packages of thinc, spacy, and confection that all were tied to specific pydantic versions.